MARIE_AND_BERT/Dockerfile_local

```diff
@@ -0,0 +1,28 @@
+FROM tiangolo/uwsgi-nginx-flask:python3.8 as flask
+
+COPY Marie/ ./Marie
+COPY uwsgi.ini ./uwsgi.ini
+COPY static/ ./static
+COPY templates/ ./templates
+COPY main.py ./main.py
+COPY requirements_linux.txt ./requirements_linux.txt
```

Comment:
As discussed, comparing this with the main `Dockerfile`, it doesn't look like the correct files/folders are being copied. Please check/fix this.

Reply:
The Dockerfile_local is updated and has been tested.